### PR TITLE
Iht 4433 Content - TNRB - Date of marriage

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/iht/controllers/application/tnrb/DateOfMarriageController.scala [hunk 1]
```diff
@@ -50,23 +60,23 @@ trait DateOfMarriageController extends EstateController{
...
               appDetails.widowCheck.fold(WidowCheck(None, None))(identity),
-              deceasedName)
+              deceasedName, predeceasedName(appDetails))
             )
-          }
           case _ => InternalServerError("Application details not found")
         }
       }
     }
   }
```
Comment (tyagikumarvineet) [inline app/iht/controllers/application/tnrb/DateOfMarriageController.scala]: Remove extra line break.

File: app/iht/views/application/tnrb/date_of_marriage.scala.html [hunk 2]
```diff
@@ -5,15 +5,14 @@
 * you may not use this file except in compliance with the License.
```
Comment (tyagikumarvineet) [inline app/iht/views/application/tnrb/date_of_marriage.scala.html]: Better to add the view test for it

Comment (grantrobson) [inline app/iht/views/application/tnrb/date_of_marriage.scala.html]: Done

File: app/iht/controllers/application/tnrb/DateOfMarriageController.scala (hunk 1 above)
Comment (grantrobson) [inline app/iht/controllers/application/tnrb/DateOfMarriageController.scala]: Done

**Pull request merged**